docs/api.rst

```diff
@@ -13,7 +13,8 @@ Infrastructure
 
 Simulation Configuration
 ------------------------
-.. autofunction:: bfrescox.Configuration
+.. autoclass:: bfrescox.Configuration
+   :members:
```

Comment:
Need to confirm that all members are rendered, including class methods, once this is merged with the fully implemented Configuration class.

We will also need to add the `parse_fort16` function here.